pkg/cqrs/cancellations.go

```diff
@@ -35,6 +35,8 @@ type CancellationWriter interface {
 
 // Cancellation represents a cancellation of many function runs during the time specified.
 type Cancellation struct {
+	CreatedAt   time.Time `json:"created_at"`
```

Comment:
CreatedBy or CreatedAt?  The name of the PR is CreatedBy 🤔

Reply:
Oops! It's CreatedAt. Messed up the PR title and description